playbooks/02 - Use Case - KEV Alert/> Notify for SLA and Report.json

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
@cs-ankit-gupta as discussed, add a note in the documentation listing out all the changes that need to be done in the playbook if BOD guidelines changes in future